src/Domains/Social/Messages/Actions/ValidateMessageSchemaAction.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\Messages\Actions;
+
+use Kanvas\Social\Messages\Models\Message;
+use Kanvas\Social\MessagesTypes\Models\MessageType;
+
+class ValidateMessageSchemaAction
```

Comment:
@rwhite27 awesome but check https://laravel.com/docs/11.x/validation to see if we can create a validation class instead of a action